### ISSUE TITLE
JavaScript Errors

### ISSUE THREAD

**Issue by rck3225** [airr-community/airr-standards#785]
There are internal JavaScript errors on the main page (Uncaught TypeError) and on v: latest (GET error). The specific pages are as follows:
https://docs.airr-community.org/en/stable/api/adc.html
https://docs.airr-community.org/en/latest/api/adc.html

**Comment by javh** [comment]
I'm not sure about the error from `stable`, but that path doesn't exist in `latest`. Do you get an error from this url?

https://docs.airr-community.org/en/latest/adc/adc.html

**Comment by rck3225** [comment]
Yes, I get an "Uncaught TypeError" and can replicate it in different
browsers. I have attached a screenshot of the error and pasted the text
below:

Uncaught TypeError: Cannot read properties of null (reading 'classList')
    at HTMLDocument.T (pydata-sphinx-theme.js:570:5)

**Comment by scharch** [comment]
cannot reproduce
